scripts/datasets/DET.py

```diff
@@ -0,0 +1,267 @@
+"""this script is used to prepare DET dataset for tracking,
+which is Object detection in Large Scale Visual Recognition Challenge 2015 (ILSVRC2015)"""
+import argparse
+import tarfile
+from os.path import join, isdir, expanduser
+from os import rename, system, chdir
+import glob
+try:
+    import xml.etree.cElementTree as ET
+except ImportError:
+    import xml.etree.ElementTree as ET
+import json
+import time
+import sys
+from concurrent import futures
+import numpy as np
+from gluoncv.utils.filesystem import try_import_cv2
+from gluoncv.utils import download, makedirs
+cv2 = try_import_cv2()
```

Comment:
Please write it inside each function or class. You can see this PR (https://github.com/dmlc/gluon-cv/pull/1158) on how to change cv2 to lazy import. We already change cv2 installation to multi-process compatible, so you shouldn't experience any trouble.